docs/learn-transactions.md

```diff
@@ -0,0 +1,62 @@
+---
+id: learn-transactions
+title: Transaction Types and Metering
+sidebar_label: Transaction Types and Metering
+---
+
+Depending on the pallets installed in a [Substrate](https://substrate.dev)-based chain like Polkadot or Kusama, different transactions are possible within the system. For example, one can propose referenda in the Governance pallet, verify identities in the Identities pallet, send tokens through the Balances pallet, shout remarks into the chain through the System pallet, and more. 
```

Comment:
```suggestion
Depending on the pallets installed in a [Substrate](https://substrate.dev)-based chain like Polkadot or Kusama, different transaction types are possible within the system. For example, one can propose referenda in the Governance pallet, verify identities in the Identities pallet, send tokens through the Balances pallet, shout remarks into the chain through the System pallet, and more. 
```

Would it be better to change the examples into a markdown list?

Reply:
I feel like that would demand more examples. This kind of prose-y exemplification lets me get away with less imo